CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `WavePort` to the `TerminalComponentModeler` which is the suggested port type for exciting transmission lines.
 - Added plotting functionality to voltage and current path integrals in the `microwave` plugin.
 - Added convenience functions `from_terminal_positions` and `from_circular_path` to simplify setup of `VoltageIntegralAxisAligned` and `CustomCurrentIntegral2D`, respectively.
+- Added axial ratio computation to `DirectivityMonitor`
```

Comment:
please mention how to grab this information, for example `DirectivityMonitor.axial_ratio`. Also what is the axial ratio? consider adding `, defined as the ratio of the major axis to the minor axis of the polarization ellipse`.

Adding this info into the changelog reduces the barrier of entry for people who just catch sight of the new feature to start using it.

otherwise PR looks good!

Reply:
also, maybe the more technically correct thing here is to say that it is added to the "`DirectivityData.axial_ratio` corresponding to a `DirectivityMonitor`".?

Reply:
Rivised, thanks @tylerflex !